proto/proto-backplane-grpc/build.gradle

```diff
@@ -18,7 +18,7 @@ configurations {
 }
 
 // TODO(deephaven-core#1685): Create strategy around updating and maintaining protoc version
-def protobufVersion = '3.17.3'
+def protobufVersion = '3.20.1'
 def boringSslVersion = '2.0.34.Final'
```

Comment:
There *may* be a specific version of boringSsl based on protobuf version... looking to see if I can find the compatibility matrix (it might have been for another dependency).

Reply:
Yep - see requirements here. https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty. We should probably add this link in the code.

Reply:
And it's based on the GRPC_VERSION - should probably extract these constants into Classpaths constants.

Reply:
Updated, PTAL.